docs/howtos/using_custom_appcharts.md

```diff
@@ -53,13 +52,29 @@ To switch a deployed application to a different Helm chart, you will have to
 delete and re-deploy the application.
 :::
 
-
 :::info
 Scaling the application to zero instances is __not sufficient__. While that effectively
 stops the application it does not remove the deployed workload, just the underlying active
 elements.
 :::
 
+# Deploying applications with a custom Helm chart which has user-settable configuration values
+
+The general mechanism is the same as described in the previous section.
+
+However, further use the option `--chart-value` to specify name and value of a single user-settable
+configuration value. Use the option multiple times to set all the desired configuration values.
+
+:::note
+User-settable configuration values are generally __optional__. The application chart is expected to
+contain and use sensible defaults for all configuration values not set by the user.
+:::
+
+Example:
+
+```
+epinio push --app-chart CHARTNAME --chart-value CVNAME=VALUE
```

Comment:
Not sure if perhaps adding name of app and path may be more clarifying to the user.
Something like:

```
epinio push  -n appname -p ~/testdir/testapp --app-chart CHARTNAME --chart-value CVNAME=VALUE
```

Reply:
Yes

Reply:
Done with commit 7115b3be85fb0438a6b20f14a45de84b2f93d5dc